tgui/packages/tgui/interfaces/GeneTek/type.ts

```diff
@@ -38,7 +38,7 @@ export interface GeneTekData {
 }
 
 interface ResearchEntry {
-  ref?: Ref;
+  ref: Ref;
   name: string;
   desc: string;
   cost?: number;
```

Comment:
If "the cost property is always set for available researches", per comment on ResearchTab.tsx:94, shouldn't this be `cost: number`? If so, can change the type and lose the comment on that line.

Reply:
No, because `ReasearchEntry`s are also used for completed researches, which don't have `cost`.